src/carbonio-files-ui-common/views/components/Displayer.unsavedChanges.editLink.test.tsx

```diff
@@ -28,93 +29,113 @@ import { formatDate, initExpirationDate } from '../../utils/utils';
+						mocks
+					});
 
-				await screen.findByText(/public link/i);
-				const link = node.links[0] as Link;
-				await screen.findByText(link.url as string);
-				await user.click(screen.getAllByRole('button', { name: /edit/i })[0]);
-				await user.click(screen.getByTestId(ICON_REGEXP.openCalendarPicker));
-				const nextMonthButton = await screen.findByRole('button', { name: /next month/i });
-				await user.click(nextMonthButton);
-				// chosen date is the 1st of next month
-				const chosenDate = getFirstOfNextMonth(link.expires_at || undefined);
-				// always click on first 1 visible on the date picker
-				await user.click(screen.getAllByText('1')[0]);
-				await screen.findByText(formatDate(chosenDate, DATE_FORMAT));
-				await user.click(screen.getByText(/details/i));
-				await screen.findByText(/you have unsaved changes/i);
-				act(() => {
-					// run timers of modal
-					jest.runOnlyPendingTimers();
-				});
-				expect(screen.getByText(/you have unsaved changes/i)).toBeVisible();
-				expect(screen.getByText(/Do you want to leave the page without saving\?/i)).toBeVisible();
-				expect(screen.getByText(/All unsaved changes will be lost/i)).toBeVisible();
-				expect(screen.getByRole('button', { name: /cancel/i })).toBeVisible();
-				expect(screen.getByRole('button', { name: /leave anyway/i })).toBeVisible();
-				expect(screen.getByRole('button', { name: /save and leave/i })).toBeVisible();
-			});
+					await screen.findByText(title);
+					const link = node.links[0] as Link;
+					await screen.findByText(link.url as string);
+					await user.click(screen.getAllByRole('button', { name: /edit/i })[0]);
+					await user.click(screen.getByTestId(ICON_REGEXP.openCalendarPicker));
+					const nextMonthButton = await screen.findByRole('button', { name: /next month/i });
+					await user.click(nextMonthButton);
+					// chosen date is the 1st of next month
+					const chosenDate = getFirstOfNextMonth(link.expires_at || undefined);
+					// always click on first 1 visible on the date picker
+					await user.click(screen.getAllByText('1')[0]);
+					await screen.findByText(formatDate(chosenDate, DATE_FORMAT));
+					await user.click(screen.getByText(/details/i));
+					await screen.findByText(/you have unsaved changes/i);
+					act(() => {
+						// run timers of modal
+						jest.runOnlyPendingTimers();
+					});
+					expect(screen.getByText(/you have unsaved changes/i)).toBeVisible();
+					expect(screen.getByText(/Do you want to leave the page without saving\?/i)).toBeVisible();
+					expect(screen.getByText(/All unsaved changes will be lost/i)).toBeVisible();
+					expect(screen.getByRole('button', { name: /cancel/i })).toBeVisible();
+					expect(screen.getByRole('button', { name: /leave anyway/i })).toBeVisible();
+					expect(screen.getByRole('button', { name: /save and leave/i })).toBeVisible();
+				}
+			);
 
 			test.skip('cancel action leaves fields valued and navigation is kept on sharing tab', async () => {
```

Comment:
Please update the remaining tests too. Remove the skip and try to run them.
The skip here was set because of a timeout, but I think we can return to run all of them now, because we have more control over the global timeout of the pipelines